imap_processing/codice/constants.py

```diff
@@ -1140,3 +1140,38 @@
         13: 180,
     },
 }
+
+# SSD ID to Elevation Angle
+SSD_ID_TO_ELEVATION = {
+    0: 150.0,
+    1: 138.6,
+    3: 115.7,
+    4: 90.0,
+    5: 64.3,
+    7: 41.4,
+    8: 30.0,
+    9: 41.4,
+    11: 64.3,
+    12: 90.0,
+    13: 115.7,
+    15: 138.6,
+}
```

Comment:
Another idea here is to turn this into a numpy index-array and insert the nan's directly in here in the indexes that are missing.
```python
SSD_ID_TO_ELEVATION = np.array([150.0, 138.6, np.nan, 115.7, 90.0, 64.3, np.nan, ...])
```